Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Profile.hs

```diff
@@ -85,7 +86,7 @@ data PersonDefaultEmergencyNumber = PersonDefaultEmergencyNumber
     mobileCountryCode :: Text,
     mobileNumber :: Text
   }
-  deriving (Generic, ToJSON, FromJSON, ToSchema)
+  deriving (Generic, ToJSON, FromJSON, ToSchema, Eq)
```

Comment:
Why do you need `Eq` instance here if you compare by `mobileNumber`?

Reply:
> Why do you need `Eq` instance here if you compare by `mobileNumber`?

removed it.